.circleci/config.yml

```diff
@@ -3,7 +3,8 @@ jobs:
   build_and_test:
     working_directory: ~/sdk
     docker:
-      - image: circleci/android:api-30
+      - image: cimg/android:2023.07
```

Comment:
Upgraded to earliest version that supports api 34

Reply:
thanks so much!